pan-domain-auth-verification/src/test/scala/com/gu/pandomainauth/PublicSettingsTest.scala

```diff
@@ -23,13 +23,13 @@ class PublicSettingsTest extends FreeSpec with Matchers with EitherValues with S
 
     "returns the key if it is valid" in {
       val key = TestKeys.testPublicKey
-      PublicSettings.validateKey(key).right.value shouldEqual key
+      PublicSettings.validateKey(key.key).right.value shouldEqual key
```

Comment:
Why not put the `.key` on `val key = TestKeys.testPublicKey` to make this a little less "keyn"? (Or have the method take a `PublicKey` which would make more sense I think)